### PR TITLE
Add additional column to Carling-2019-99, rename concept list

### PR DESCRIPTION
**Pull request by chrzyki**
# Pull request checklist

- [ ] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
- [ ] add new Concepticon concept relations
- [x] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [x] retire data

## Additional information

This adds the `CLASSIFICATION` column as outlined in #905. It also renames the list because the last item (`TOTAL`) was, as far as I can tell, not part of the original list but an artifact of the sheet layout.

### CODE REVIEW

File: concepticondata/retired.json [hunk 1]
```diff
@@ -28779,6 +28779,506 @@
...
+    },
+    {
+      "id": "Carling-2019-100-99",
+      "comment": "renaming",
+      "replacement": "Carling-2019-99-99"
+    },
+    {
+      "id": "Carling-2019-100-100",
+      "comment": "renaming",
+      "replacement": "Carling-2019-99-100"
```
Comment (Kristina-Pianykh) [inline concepticondata/retired.json:29281]: Has the policy of retiring concepts that are not in the list changed? I think I used `"replacement": null `once but as far as I can tell it's already outdated.

Comment (chrzyki) [reply concepticondata/retired.json:29281]: Nope, I was just missing a commit. Thanks.

**Comment by Kristina-Pianykh** [comment]
The data they provided contain 99 concepts. I honestly have no idea where the 100th concept TOTAL came from in the first version of the list. Here's what the authors says o the semantic categories:

> Our data includes words for predator and game animals, domestic animals, farming, metallurgy, tools, weapons, implements, and so forth. The basis for our list is a selection of core meanings, which are of primary importance to subsistence and economy (see Table 1), and which can be reconstructed to the proto-languages of the families, in particular Indo-European [18, 19]. We have extracted the dataset from the lexical subsection of the database DiACL [20], which uses a taxonomic classification of concepts (including activities, agricultural tools, crops & fruit, farm animals, products, seasons, trees, weapons, wild animals, etc.) based on the standard of the Intercontinental Dictionary Series [4, 21]. For the current study, we use a semantic classification of our data that is based on clustering of colexifications (co-occurrence of meanings of concepts) and meaning changes [22].

**Comment by chrzyki** [comment]
The list was already released so retiring is necessary. Categories come from this supplement: https://doi.org/10.1371/journal.pone.0223588.s003.

The 'TOTAL' came from one of the supplements where it was the last row, being the TOTAL of the cells but not an individual row/concept.

**Pull request merged**